src/components/Settings.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 Spotify AB
```

Comment:
Spotify are nice guys and all but copyright RoadieHQ please

Reply:
Done